python/samples/concepts/prompt_templates/load_yaml_prompt.py

```diff
@@ -13,15 +13,18 @@ async def main():
 
     service_id = "default"
     chat_service = OpenAIChatCompletion(
-        ai_model_id="gpt-4-0613",
+        ai_model_id="gpt-3.5-turbo-1106",
```

Comment:
We want this older model instead of the newer one?

Reply:
No need for a powerful model for this, and isn't gpt-3.5 cheaper? :)